pkg/appmeshinject/config.go

```diff
@@ -0,0 +1,107 @@
+package appmeshinject
+
+import (
+	"errors"
+	"flag"
+)
+
+type Config struct {
+	// Injetion Settings
+	InjectDefault bool
+
+	// If enabled, an fsGroup: 1337 will be injected in the absence of it within pod securityContext
+	// see https://github.com/aws/amazon-eks-pod-identity-webhook/issues/8 for more details
+	EnableIAMForServiceAccounts bool
+
+	// Sidecar settings
+	SidecarImage  string
+	SidecarCpu    string
+	SidecarMemory string
+	//MeshName      string
+	Region    string
+	Preview   bool
+	LogLevel  string
+	EcrSecret bool
+
+	// Init container settings
+	InitImage          string
+	IgnoredIPs         string
+	EgressIgnoredPorts string
+
+	// Observability settings
+	InjectXraySidecar           bool
+	EnableStatsTags             bool
+	EnableStatsD                bool
+	InjectStatsDExporterSidecar bool
+	EnableJaegerTracing         bool
+	JaegerAddress               string
+	JaegerPort                  string
+	EnableDatadogTracing        bool
+	DatadogAddress              string
+	DatadogPort                 string
+}
+
+// MultipleTracer checks if more than one tracer is configured.
+func MultipleTracer(config *Config) bool {
+	j := config.EnableJaegerTracing
+	d := config.EnableDatadogTracing
+	x := config.InjectXraySidecar
```

Comment:
nit: since we rewrite this, it's a chance for us to normalize these flags. e.g. use InjectXXX or EnableXXX consistently.
I think we can also do it after merge this.

Reply:
will follow up